homeworks/06-advanced-redux/src/App.tsx

```diff
@@ -12,21 +12,31 @@ import {
   TotalPrice,
 } from './components';
 
-function App() {
-  const {
-    addItemToCart,
-    cart,
-    menu,
-    removeItemFromCart,
-    totalPrice,
-  } = useApp();
+import { shop } from './redux/store';
+import { addToCart, removeFromCart, selectItem } from './redux/actions/shop';
+import { loadMenuAsync } from './redux/actions/menu';
+
+import { Pizza, ShopState } from './types';
+
+interface AppProps {
+  state: ShopState
+}
+
+function App(props: AppProps) {
+  const { menu, cart } = props.state;
```

Comment:
there is not really need it because you are able to do it within `mapStateToProps`

Reply:
Updated to use useSelector, see https://github.com/vbarbasins/evo-ts-bootcamp/pull/7/commits/40000cf19231ba1b304ed01ab3879930296ac2a9